README.md

```diff
@@ -16,8 +16,7 @@
 4. have consistent uptime.
    - Minor downtime for maintenance or fixing issues is fine
 5. have bot listing requirements (such as no NSFW outside NSFW channels, respect rate-limits, don't respond to other bots, etc.)
-6. blur NSFW profile pictures.
```

Comment:
I personally suggest to instead of removing this, mention to NOT have this.
Like `6. not allow bots with NSFW images.`

Alternatively could this be added as an example to point 5